easybuild/easyconfigs/p/pkgconfig/pkgconfig-1.2.2-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'pkgconfig'
+version = '1.2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://github.com/matze/pkgconfig'
+description = """pkgconfig is a Python module to interface with the pkg-config command line tool"""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
```

Comment:
@jhein32 please include a (SHA256) checksum here too, as well as in the existing easyconfig that uses `intel/2017a`?